### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (112)
Screenshot (116)

PLEASE REVIEW ONLY SIGNUP AND LOGIN PAGE!!

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Good UI design and folder structure. Would suggest converting html to jsx either manually 
(class -> className, for -> htmlFor etc.) or using some conversion tool like https://transform.tools/html-to-jsx

File: src/public-css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.container {
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+}
+
+.search-box {
+    width: 28rem;
+    height: 3rem;
+    border: 1px solid var(--secondary-color);
+    border-radius: 1.8rem;
+    padding: 10px;
```
Comment (devAmlan) [inline src/public-css/navbar.css:10]: use rem here

File: src/cave-components/Wishlist/wishlist.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                <a href="/index.html">SPORTS CAVE</a>
+            </div>
+            <div>
+                <input type="text" placeholder="Search" name="search" class="search-box" />
+            </div>
+            <div class="right-nav">
+                <button class="btn-secondary-outline btn btn-text no-margin btn-round"><a href="/index.html" class="login-color">Logout</a></button>
+                <div class="badge-item">
+                    <button class="btn btn-only-icon no-margin">
+                        <a href="./wishlist.html" class="items-color"><i class="far fa-heart fa-2x"></i></a></button>
```
Comment (devAmlan) [inline src/cave-components/Wishlist/wishlist.jsx:21]: change the html to jsx code

**Review by devAmlan — APPROVED** [review]
1. Code looks good
2. Change your html codes to jsx.
3. Suggested some changes

Good to merge

File: src/cave-components/Wishlist/wishlist.jsx (hunk 2 above)
Comment (Megha-Joshi) [reply src/cave-components/Wishlist/wishlist.jsx:21]: Okay

File: src/public-css/navbar.css (hunk 1 above)
Comment (Megha-Joshi) [reply src/public-css/navbar.css:10]: Okay

**Pull request merged**